gradle-with-plugin/.gitignore

```diff
@@ -0,0 +1,16 @@
+# Eclipse
```

Comment:
Is it necessary to copy this .gitignore file from the root?
Or is it for easily starting your project by copy pasting a subdirectory?

Reply:
That was indeed the idea...